rules/xcodeproj.bzl

```diff
@@ -31,7 +31,6 @@ def _xcodeproj_aspect_impl(target, ctx):
     test_commandline_args = ()
     if AppleBundleInfo in target:
```

Comment:
I am trying to understand this if stmt because the code change applies to all blocks of this if stmts correct?
there are 
1. `AppleBundleInfo in target`
2. `ctx.rule.kind == "apple_framework_packaging"`
3. `else`
I wonder each is for what cases?

Reply:
In practice:
1. `if AppleBundleInfo in target` is generally applied to targets that are unit test bundles or application bundles.
2. When the target is packaged up framework (this target usually depends on the components of a framework, like libraries, resource bundles, plists.)
3. Everything else. If the target is not performing a bundling/packaging operation, it often contains some kind of source files and dependencies with their source files.